nixos/modules/misc/ids.nix

```diff
@@ -350,6 +350,7 @@ in
       # shadow = 318; # unused
       hqplayer = 319;
       moonraker = 320;
+      distcc = 321;
```

Comment:
Can't we use dynamic user?

Reply:
No, otherwise you can't combine it with ccache, sadly.